db/structure.sql

```diff
@@ -14,55 +7,6 @@ SET check_function_bodies = false;
 SET client_min_messages = warning;
 SET row_security = off;
 
---
--- Name: revamp; Type: SCHEMA; Schema: -; Owner: -
---
-
-CREATE SCHEMA revamp;
```

Comment:
All of `revamp` schema gets removed when you run `rake db:migrate`, sadly because of the search path set to `public` on the default database connection. To avoid issues with messing up the schema temporarily comment out the search path setting in `database.yml` when running migrations.